### ISSUE TITLE
[Android] ToolbarAndroid is not accessible

### ISSUE THREAD

**Issue by oargaruna** [facebook/react-native#12370]
### Description

There is no prop in ToolbarAndroid which allows us to provide accessibility labels for the navigation icon. Ideally, we should be able to provide either an image source, or a View as the nav icon.

### Reproduction

1. Add a ToolbarAndroid component with a nav icon to any React Native View.
2. Activate Talkback on an Android phone.
3. Bring Talkback focus to the nav icon.
4. Talkback reads it as "Button unlabelled".

### Additional Information

* React Native version: 0.41
* Platform: Android

**Comment by hramos** [comment]
We're cutting down on the number of outstanding issues, in order to allow us to focus. I'm closing this issue because it has been open for over 60 days with no activity. If you think it should still be opened let us know why. PRs are always welcome.
